specs/core/1_shard-data-chains.md

```diff
@@ -137,7 +145,8 @@ def get_period_committee(state: BeaconState, epoch: Epoch, shard: Shard, index:
 ```python
 def get_switchover_epoch(state: BeaconState, epoch: Epoch, index: ValidatorIndex):
     earlier_start_epoch = epoch - (epoch % PERSISTENT_COMMITTEE_PERIOD) - PERSISTENT_COMMITTEE_PERIOD * 2
-    return bytes_to_int(hash(generate_seed(state, earlier_start_epoch) + bytes3(index))[0:8]) % PERSISTENT_COMMITTEE_PERIOD
```

Comment:
I am not exactly sure what is meant by `bytes3(index))[0:8]` here. I changed it to `int_to_bytes(index, length=3)[0:8]` as this seemed the most intuitive to me, but slicing bytes of length 3 into [0:8] doesn't make much sense.